src/python/ddapp/roboturdf.py

```diff
@@ -30,6 +30,7 @@
     handCombinations = directorConfig['handCombinations']
     numberOfHands = len(handCombinations)
 
+    headLink = directorConfig['headLink']
```

Comment:
This willl break config files that do not contain headLink.  I think that's ok.  That is how we have been operating so far.  Eventually we should talk about what must be in a config file and what is optional.  For example, to make this optional, we could say:

    headLink = directorConfig.get('headLink', '') # default to empty string if not defined by config file